pyanaconda/core/configuration/storage.py

```diff
@@ -87,6 +98,31 @@ def default_partitioning(self):
         """
         return self._get_option("default_partitioning", PartitioningType)
 
+    @property
+    def default_scheme(self):
+        """Default partitioning scheme.
+
+        Valid values:
+
+          0  PLAIN      Create standard partitions.
+          1  BTRFS      Use the Btrfs scheme.
+          2  LVM        Use the LVM scheme.
+          3  LVM_THINP  Use LVM Thin Provisioning.
+
+        :return: a partitioning scheme
+        """
+        return self._get_option("default_scheme", self._convert_partitioning_scheme)
+
+    def _convert_partitioning_scheme(self, value):
+        """Convert the given value into a partitioning scheme."""
+        try:
+            member = PartitioningScheme.__members__[value]
+            return member.value
+        except KeyError:
+            pass
+
+        raise ValueError("'{}' is not a valid partitioning scheme".format(value))
```

Comment:
Please move this into the enum as method. I don't think we should access ``__member__`` private member out of the class.

Reply:
Fixed.